.github/workflows/main.yml

```
@@ -20,7 +20,7 @@ jobs:
           node-version: ${{ matrix.node }}
 
       - name: Use cached node_modules
-        uses: actions/cache@v2
+        uses: actions/cache@v2.1.4
```

Using the major version will create less PRs

```suggestion
        uses: actions/cache@v2
```